trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/is-pdc-reconcile-user.ts

```diff
@@ -0,0 +1,10 @@
+import { TEAM_IDS } from '@ukef/dtfs2-common';
+import { TfmSessionUser } from '../../../types/tfm-session-user';
+
+/**
+ * isPDCReconcileUser
```

Comment:
Can we remove this line please, just repeating the function name doesn't add any value since this will always be read in conjunction with the function name

Reply:
removed